packages/styles/src/generic-tag.scss

```diff
@@ -0,0 +1,151 @@
+      "background": var(--sapSuccessBackground),
+      "value-hover": var(--fdGenericTag_Value_Color_Hover_Success),
+      "icon-hover": var(--fdGenericTag_Icon_Color_Hover_Success),
+    ),
+    "information": (
+      "value": var(--sapInformativeTextColor),
+      "icon": var(--sapInformativeElementColor),
+      "decorator": var(--sapInformativeElementColor),
+      "background": var(--sapInformationBackground),
+      "value-hover": var(--fdGenericTag_Value_Color_Hover_Information),
+      "icon-hover": var(--fdGenericTag_Icon_Color_Hover_Information),
+    )
+  );
+
+  @include fd-reset();
+  @include fd-flex-vertical-center();
+  @include fd-set-paddings-x(0.75rem, 0.375rem);
+
+  cursor: pointer;
+  height: 1.625rem;
+  width: fit-content;
+  gap: var(--fdGenericTag_Spacing);
+  margin: var(--fdGenericTag_Margin);
+  box-shadow: var(--sapContent_Shadow0);
+  border-radius: var(--sapButton_BorderCornerRadius);
+  background: var(--fdGenericTag_Decorator_Position) 0 / var(--fdGenericTag_Spacing) 100% no-repeat linear-gradient(0deg, var(--fdGenericTag_Decorator_Color), var(--fdGenericTag_Decorator_Color));
+  background-color: var(--fdGenericTag_Background_Color);
+
+  &__icon[class*=sap-icon] {
+    @include fd-flex-center();
+
+    font-size: 1rem;
+    color: var(--fdGenericTag_Icon_Color);
+  }
+
+  &__name {
+    @include fd-reset();
+    @include fd-ellipsis();
+
+    color: var(--fdGenericTag_Name_Color);
+    min-width: var(--fdGenericTag_Text_Min_Width);
+    text-shadow: var(--fdGenericTag_Name_Text_Shadow);
+  }
+
+  &__value {
+    @include fd-reset();
+    @include fd-ellipsis();
+    @include fd-set-margin-left(0.75rem);
+
+    color: var(--fdGenericTag_Value_Color);
+    min-width: var(--fdGenericTag_Text_Min_Width);
+    text-shadow: var(--fdGenericTag_Value_Text_Shadow);
+  }
+
+  // STATES
+  @include fd-hover() {
+    --fdGenericTag_Background_Color: var(--sapList_Hover_Background);
+    --fdGenericTag_Name_Color: var(--fdGenericTag_Name_Color_Hover);
+    --fdGenericTag_Value_Color: var(--fdGenericTag_Value_Color_Hover_Neutral);
+    --fdGenericTag_Icon_Color: var(--fdGenericTag_Icon_Color_Hover_Neutral);
+    --fdGenericTag_Name_Text_Shadow: none;
+    --fdGenericTag_Value_Text_Shadow: none;
+  }
+
+  @include fd-active() {
+    --fdGenericTag_Background_Color: var(--sapList_Active_Background);
+    --fdGenericTag_Name_Color: var(--sapList_Active_TextColor);
+    --fdGenericTag_Name_Text_Shadow: none;
+    --fdGenericTag_Icon_Color: var(--sapList_Active_TextColor);
+    --fdGenericTag_Value_Color: var(--sapList_Active_TextColor);
+    --fdGenericTag_Value_Text_Shadow: none;
+  }
+
+  @include fd-focus() {
+    outline-offset: var(--fdGenericTag_Spacing_Outline_Offset);
+    outline: var(--sapContent_FocusColor) var(--sapContent_FocusStyle) var(--sapContent_FocusWidth);
+  }
+
+  @include fd-rtl() {
+    --fdGenericTag_Decorator_Position: 100%;
+  }
+
+  // TYPES
+  &.#{$block}--compact {
```

Comment:
Please use `fd-compact-and-condensed` mixin

Reply:
Done